### ISSUE TITLE
Deferred FxA deeplinking sign-in codes are not consumed at expected rate [bz1404089]

### ISSUE THREAD

**Issue by bug-mirror** [mozilla/fxa-bugzilla-mirror#406]
From https://bugzilla.mozilla.org/show_bug.cgi?id=1404089

When compared to Android FxA deferred deeplinking, sign-in codes have a significantly lower conversion rate on iOS. We expected a similar rate of conversion between the platforms.

This difference could be a result of some invalid metrics and or something broken in while processing the code. We should examine why this is not working as expected.

**Comment by vbudhram** [comment]
Round 3 of getting adjust links working as expected in iOS, https://github.com/mozilla-mobile/firefox-ios/pull/3438.

**Comment by vbudhram** [comment]
PR merged, blocked on 10.2 being released.

**Comment by vladikoff** [comment]
from mtg: waiting for ios 10.2

**Comment by vladikoff** [comment]
from mtg: 10.3 released, to check the graphs!

**Comment by vladikoff** [comment]
from mtg: @vbudhram to check graphs?

**Comment by vbudhram** [comment]
Latest dashboards still do not show an improvement in rates, https://analytics.amplitude.com/org/31251/dashboard/or8kj3j.

*However*, codes dashboards show that sign-in codes are appearing on iOS FF devices. Will follow up with Adjust support on latest findings.

**Comment by vladikoff** [comment]
from mtg: Adjust says gave some stuff to try...
